mango-core/src/main/java/org/calrissian/mango/domain/AttributeStore.java

```diff
@@ -47,6 +37,9 @@
      */
     <T> Attribute<T> get(String key);
 
+    <T> Attribute<T> get(String key, Class<T> clazz);
```

Comment:
On a side note, since this is a multivalued data object, shouldn't the get methods return collections?